gcp/appengine/blog/hugo.toml

```diff
@@ -1,3 +1,5 @@
 baseURL = 'http://osv.dev/blog/'
 languageCode = 'en-us'
 title = 'OSV Blog'
+[markup.goldmark.renderer]
+unsafe= true
```

Comment:
This is needed for hugo to render raw html. It should be ok because we don't have any user-submitted html. https://discourse.gohugo.io/t/raw-html-getting-omitted-in-0-60-0/22032